samples/bluetooth/hci_spi/README.rst

```diff
@@ -0,0 +1,46 @@
+Bluetooth: HCI SPI
+##################
+
+Overview
+********
+
+Expose Zephyr Bluetooth Controller support over SPI to another device/CPU using
+the Zephyr SPI HCI transport protocol (similar to BlueNRG).
+
+Requirements
+************
+
+* A board with SPI slave, GPIO and BLE support
```

Comment:
I'll add a period, but since this is not a full sentence, I think it's up to style whether it's "missing".